ReactNativeClient/lib/services/CommandService.ts

```diff
@@ -75,8 +76,11 @@ export default class CommandService extends BaseService {
 	private commandPreviousStates_:CommandStates = {};
 	private mapStateToPropsIID_:any = null;
 
-	initialize(store:any) {
+	private keymapService:typeof KeymapService = null;
+
+	initialize(store:any, keymapService:typeof KeymapService) {
```

Comment:
Once you've updated the import statement, the "typeof KeymapService" should no longer be needed, and you can just do `keymapService:KeymapService`

Reply:
Thanks for pointing that out. I'll update it right away.